src/schema/sale_artwork.js

```diff
@@ -272,6 +261,42 @@ const SaleArtworkType = new GraphQLObjectType({
         type: GraphQLFloat,
         deprecationReason: "Favor `minimum_next_bid`",
       },
+      my_increments: {
```

Comment:
I don't have a problem with this naming, esp since there's already an `increments` schema here. Maybe that should accept an arg `useMyMaxBid: true/false` to perform this more nuanced behavior and keep things contained to that field?

Reply:
The `useMyMaxBid` param is a good idea too. Let me spike out what that would look like.